tests/pii_replacer/test_patterns.py

```
@@ -0,0 +1,753 @@
+                ],
+            )
+        ],
+        standalone_columns_to_replace=[
+            PiiColumnPlan(column_name="date_of_birth", entity_type=PiiEntity.date_of_birth, patterns=patterns),
+        ],
+    )
+    runtime = Config(
+        locale="en_US",
+        random_seed=7,
+        persona_backend="faker",
+        sdg_pgms_src="/tmp",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)